### PR TITLE
Adding keySize functionality for MultiMaps

### PR DESCRIPTION
**Pull request by mdheeraj24**
Adding a new functionality which gets the count of all the distinct keys for a multimap.

### CODE REVIEW

**Comment by mrniko** [comment]
`map.keySet().size()` should be used instead, but it seems it doesn't work. I will fix it.

File: src/main/java/org/redisson/RedissonListMultimap.java [hunk 1]
```diff
@@ -65,6 +65,19 @@
                 "return size; ",
                 Arrays.<Object>asList(getName()));
     }
+    
+    public Future<Integer> keySizeAsync() {
+    	return commandExecutor.evalReadAsync(getName(), codec, RedisCommands.EVAL_INTEGER,
```
Comment (mrniko) [inline src/main/java/org/redisson/RedissonListMultimap.java]: Please replace it to `return commandExecutor.readAsync(getName(), LongCodec.INSTANCE, RedisCommands.HLEN, getName());` instead due to performance gain

File: src/main/java/org/redisson/RedissonSetMultimap.java [hunk 2]
```diff
@@ -69,6 +69,19 @@
                 Arrays.<Object>asList(getName()));
     }
 
+    public Future<Integer> keySizeAsync() {
+    	return commandExecutor.evalReadAsync(getName(), codec, RedisCommands.EVAL_INTEGER,
```
Comment (mrniko) [inline src/main/java/org/redisson/RedissonSetMultimap.java]: Please replace it to `return commandExecutor.readAsync(getName(), LongCodec.INSTANCE, RedisCommands.HLEN, getName());` instead due to performance gain

**Comment by mrniko** [comment]
@mdheeraj24 I have commented your code. Also please move `keySize` and `keySizeAsync` methods to `RedissonMultimap` object. Because it's independent of Multimap values structures RListMultimap and RSetMultimap

**Comment by mdheeraj24** [comment]
Would rather not prefer getting the keySet and calling a size() on it.

**Comment by mrniko** [comment]
Thank you!

**Comment by mdheeraj24** [comment]
Thank you! When can we expect the tentative date for 2.2.17 ?

**Comment by mrniko** [comment]
in two weeks I think

**Pull request merged**